migrate/migrate.md

```diff
@@ -0,0 +1,127 @@
+# Migration Tool
+Gravwell provides an interactive tool for migrating text files and Splunk data into Gravwell. This document describes how to install, configure, and use it.
+
+## Installation
+
+As the migration tool is still in early development, you’ll need to compile it yourself (or contact Gravwell for a binary file). Make sure you have [Go](https://golang.org) installed, then check out the repository and build the tool:
```

Comment:
Its released, not early development.  Just show how to build it, we probably want to talk about building a "tools" package that delivers these into `/usr/local/sbin/gravwell_****`